solc/CommandLineInterface.cpp

```diff
@@ -931,45 +932,61 @@ void CommandLineInterface::link()
 		// be just the cropped or '_'-padded library name, but this changed to
```

Comment:
I don't think we need this comment any more, at least not at this point in the code.

Reply:
I thought it was still relevant but looking closer at the PR now, it removes support for old-style placeholders and without it it's not necessary.

Are we fine with that though? Modern compiler versions do not produce them so I'd be fine with that but it's technically a breaking change.